src/poller/poller-core/index.js

```diff
@@ -304,6 +314,7 @@ async function getSpannerMetadata(projectId, spannerInstanceId, units) {
       regional: !!metadata['config']?.split('/')?.pop()?.startsWith('regional'),
       // DEPRECATED
       currentNodes: assertDefined(metadata['nodeCount']),
+      numDatabases: numDatabases,
```

Comment:
For clarity perhaps reformat these few lines or move the comment to show that it is only the `currentNodes` field that is deprecated (despite the single-line comment).

Reply:
Done